### ISSUE TITLE
Errors fails silently in setup.py

### ISSUE THREAD

**Issue by William-Hill** [CDAT/jupyter-vcdat#36]
**Describe the bug**
Errors do not stop script execution when invoking the the runme() function.  I'm currently getting an error when running `runme("jupyter labextension install .")`, but the error message is buried in stdout and continues execution

**To Reproduce**
1. Do the prequesite steps in the README to create the conda env and install dependencies
2. Run python setup.py install
3. Look for error message during the 'Running:  jupyter labextension install .' step.

**Expected behavior**
For an error to be raised and execution to stop when an error occurred.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - High Sierra 10.13.6

**Environment Information**
<details open><summary><code>`conda info`</code></summary><p>

```
active environment : jupyter-vcdat
    active env location : /anaconda2/envs/jupyter-vcdat
            shell level : 2
       user config file : /Users/hill119/.condarc
 populated config files : /Users/hill119/.condarc
          conda version : 4.5.12
    conda-build version : 3.17.6
         python version : 2.7.15.final.0
       base environment : /anaconda2  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/osx-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /anaconda2/pkgs
                          /Users/hill119/.conda/pkgs
       envs directories : /anaconda2/envs
                          /Users/hill119/.conda/envs
               platform : osx-64
             user-agent : conda/4.5.12 requests/2.21.0 CPython/2.7.15 Darwin/17.7.0 OSX/10.13.6
                UID:GID : 47256:47256
             netrc file : /Users/hill119/.netrc
           offline mode : False



```
</p></details>

**Additional context**

**Comment by doutriaux1** [comment]
@William-Hill I ran the instruction on @downiec branhc: `variable_selections` on my mac, and I ddin't have any issue. Could you try to update your conda and clean you cached packages
```
conda update --all -n base
conda clean --all -y
```

thanks.

**Comment by William-Hill** [comment]
Resolved by #37
